bibliography.bib

```diff
@@ -7142,3 +7142,694 @@ @Article {ProHsiKum2017IEEETR
+}
+
+@InProceedings{HanAndMad-etal2020sii,
+    title        = {Transferring human manipulation knowledge to robots with inverse reinforcement learning},
+    author       = Hansen_EB # and # Andersen_RE # and # Madsen_S # and # Bogh_S,
+    booktitle    = sii_2020,
+    editor       = {},
+    series       = {},
+    volume       = {35},
+    number       = {12},
+    year         = {2020},
+    pages        = {933--937},
+    publisher    = pub_IEEE,
+    address      = {},
+    doi          = {10.1109/SII46433.2020.9025873},
+    note         = {},
+    annote       = {},
+}
+
+@Article{OsaPajNeu-etal2018ftr,
+    title        = {An algorithmic perspective on imitation learning},
+    author       = Osa_T # and # Pajarinen_J # and # Neumann_G # and # Bagnell_JA # and # Abbeel_P # and # Peters_J,
+    journal      = FTR,
+    series       = {},
+    volume       = {7},
+    number       = {1-2},
+    year         = {2018},
+    pages        = {1--179},
+    doi          = {10.1561/2300000053},
+    note         = {},
+    annote       = {},
+}
+
+@InProceedings{NatKunJud-etal2010icmla,
+    title        = {Multi-agent inverse reinforcement learning},
+    author       = Natarajan_S # and # Kunapuli_G # and # Judah_K # and # Tadepalli_P # and # Kersting_K # and # Shavlik_J,
+    booktitle    = icmla_2010,
+    editor       = {},
+    series       = {},
+    volume       = {35},
+    number       = {12},
+    year         = {2010},
+    pages        = {395--400},
+    publisher    = pub_IEEE,
+    address      = {},
+    doi          = {10.1109/ICMLA.2010.65},
+    note         = {},
+    annote       = {},
+}
+
+@Article{Mah1996ml,
+    title        = {Average reward reinforcement learning: foundations, algorithms, and empirical results},
+    author       = Mahadevan_S,
+    journal      = ML,
+    series       = {},
+    volume       = {22},
+    number       = {1},
+    year         = {1996},
+    pages        = {159--195},
+    doi          = {10.1023/A:1018064306595},
+    note         = {},
+    annote       = {},
+}
+
+@Article{SonRenSad-etal2018anips,
+    title        = {Multi-agent generative adversarial imitation learning},
+    author       = Song_J # and # Ren_H # and # Sadigh_D # and # Ermon_S,
+    journal      = ANIPS,
+    series       = {},
+    volume       = {31},
+    number       = {},
+    year         = {2018},
+    pages        = {},
+    doi          = {},
+    note         = {},
+    annote       = {},
+}
+
+@Article{SosKhuZou-etal2016arxiv,
+    title        = {Inverse reinforcement learning in swarm systems},
+    author       = Sosic_A # and # KhudaBukhsh_WR # and # Zoubir_AM # and # Koeppl_H,
+    journal      = {arXiv preprint arXiv:1602.05450},
+    series       = {},
+    volume       = {},
+    number       = {},
+    year         = {2016},
+    pages        = {},
+    doi          = {10.48550/arXiv.1602.05450},
+    note         = {},
+    annote       = {},
+}
+
+@Article{BahGupPat2022arxiv,
```

Comment:
see https://github.com/demiurge-project/demiurge-bib/blob/5c51777aae7c3ef2292759a85850f7395276e218/bibliography.bib#L2822

for arxiv papers